src/build/get_xcodebuild_destination.mjs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import os from 'node:os';
+import {getIosDeploymentTarget} from './get_ios_deployment_target.mjs';
+
+// https://mokacoding.com/blog/xcodebuild-destination-options/
+export async function getXcodebuildDestination(platform) {
```

Comment:
I think this is ever only needed in the test? Perhaps we should call it `getXcodebuildTestDestination`?
However, I think it would be more readable if you just inlined the logic in the test action.